.env

```diff
@@ -0,0 +1,4 @@
+ REACT_APP_BACKEND_URL= 'http://localhost:3040',
+ ADMIN_EMAIL= "drec@energy.org",
+ ADMIN_PASSWORD= "1234Rc",
+ UI_BASE_URL= 'http://localhost:4200'
```

Comment:
can we put this `.env` elsewhere? I don't want the production instance to use these values

Reply:
I think we don't need this file

Reply:
I hardcoded the variables because I don’t have permission to add secrets. If someone can add them under Settings > Secrets, I’ll update the code to use those instead.